cordova-plugin-appcenter-shared/src/android/AppCenterShared.java

```diff
@@ -29,6 +31,13 @@ static void configureAppCenter(Application application, CordovaPreferences prefe
         AppCenter.setWrapperSdk(wrapperSdk);
         AppCenter.configure(application, AppCenterShared.getAppSecret(preferences));
 
+        final int logLevel = preferences.getInteger(LOG_LEVEL_KEY, Integer.MIN_VALUE);
```

Comment:
What do you mean by persisted? It is stored in the config file, so that we set the log level before the start. This line reads the value from `config.xml`

Reply:
Maybe misreading `preference` to `persistence`? 😄

Reply:
not misreading, it's definitely persisted in config.xml ;)